1_Installation_Files (v1.5.5)/YOUR_ADMIN/image_handler.php

```diff
@@ -319,13 +319,14 @@ function popupWindow(url) {
             <td class="ih-center"><?php echo zen_draw_products_pull_down('products_filter', 'size="5"', '', true, $products_filter, true, true); ?></td>
             <td id="ih-p-buttons" class="ih-center ih-vtop">
 <?php 
-        echo zen_image_submit('button_display.gif', IMAGE_DISPLAY) . '<br />';
-        
+        //echo zen_image_submit('button_display.gif', IMAGE_DISPLAY) . '<br />';
```

Comment:
Sorry, missed this one.  Remove the comment and use the `IMAGE_DISPLAY` constant instead of a hard-coded `display` on the next line.  Also indent the `echo` so that it's in-line with the other code.